### DISCUSSION TITLE
keyball61　トラックボール無し側の左使用

### DISCUSSION THREAD

**Discussion by yummyKB** [Yowkees/keyball#125] [Q&A] [answered]
昨日届いたkeyall61のトラックボール無し側を左として組み立て見ましが
5のキーを入力すると2のキーとして反応します。
（2のキーは2のキーとして反応）

これを治す方法はありますでしょうか？

因みに今回のこのPCBですが、一緒に購入したLEDがとても入りづらいかったです。
全て力任せで押しこんでしまいましたが、トラックボール側のPCBはガタガタ
なので穴の出来がちょっと悪そうですね。とりあえず全てのLEDをハンダ付し
ましたが1 個しか点灯させることができませんでした。
まぁ、LEDは必須ではないので楽しんで修正していきたいです。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

2キーが誤って入力される件についてはキーに対応するダイオードの方向が逆である可能性が濃厚です。5キー周辺（もしくは2キー周辺）のダイオードの極性チェックをお願いします。

次にLED穴についてですが、製造データは変更していませんが、現在出荷中のロットにドリル加工の甘いものが混入したようで申し訳ありません。先日手持ちの基板は目視検査してキツそうなものは弾かせていただいております。

ちなみにLED穴がキツもので、私もはんだ付けしてみまして、挿入に力がいりますが電気的には大丈夫なようで、点灯はできました。
もしご希望でしたら、新しい基板と交換させていただきますのでお願いいたします。

**Reply by yummyKB** [reply]
Yowkeesさん、回答ありがとうございます。

ダイオードの極性についてはチェックして間違っていたのが確認できたので早速付け替えて正常動作が確認できました。

ありがとうございました。

LED穴については一旦は全てLEDの点灯が確認できたので、後はぼちぼち弄って楽しみたいと思います。

こちらもありがとうございました。
